src/new-client/testing/global-setup.ts

```diff
@@ -0,0 +1,55 @@
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+import env from './testEnvironment';
+import { promises as fs } from 'fs';
+import * as path from 'path';
+import { CosmosClient } from '@azure/cosmos';
+import { type FullConfig } from '@playwright/test';
+
+const { COSMOS_ENDPOINT, COSMOS_TOKEN, SEED_COSMOS_BEFORE_RUN } = env;
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+async function globalSetup(config: FullConfig) {
+  if (SEED_COSMOS_BEFORE_RUN) {
+    await seedCosmos();
+  }
```

Comment:
@kicr1 I put in this config value because I know the pipeline runs the .NET version so we may not want to run it again here